src/core/src/main/java/org/apache/jmeter/util/SSLManager.java

```diff
@@ -143,7 +143,12 @@ protected synchronized JmeterKeyStore getKeyStore() {
               } else {
                  File initStore = new File(fileName);
                  if (fileName.length() > 0 && initStore.exists()) {
-                     retryLoadKeys(initStore, true);
+                     String os = System.getProperty("os.name");
+                     if ( os.equals("z/OS") ) {
```

Comment:
Is it worth naming the parameter e.g.
```
boolean isZOS = System.getProperty("os.name").equals("z/OS");
boolean allowEmptyPassword = !isZOS
retryLoadKeys(initStore, allowEmptyPassword);
```

Reply:
Sounds like a plan. Then others can later also add to the boolean allowEmptyPassword easily.